core/src/main/scala/org/apache/spark/SparkContext.scala

```diff
@@ -1646,7 +1646,7 @@ class SparkContext(config: SparkConf) extends Logging {
   def killExecutors(executorIds: Seq[String]): Boolean = {
     schedulerBackend match {
       case b: CoarseGrainedSchedulerBackend =>
-        b.killExecutors(executorIds, replace = false, force = true).nonEmpty
+        b.killExecutors(executorIds, replace = false, force = false).nonEmpty
```

Comment:
Just found out that SPARK-16554 just fixed the issue of killing busy executors. Reverted this change and updated the patch. Thanks.